sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/AuthTokenContext.java

```diff
@@ -7,15 +7,24 @@
 
```

Comment:
It looks like this file comes from another PR. Please rebase to `feature/identity`